libminifi/include/utils/file/FileUtils.h

```diff
@@ -303,55 +322,74 @@ inline bool get_uid_gid(const std::string &path, uint64_t &uid, uint64_t &gid) {
 #endif
 
 inline int is_directory(const char * path) {
-    struct stat dir_stat;
-    if (stat(path, &dir_stat) < 0) {
-        return 0;
-    }
-    return S_ISDIR(dir_stat.st_mode);
+  struct stat dir_stat;
+  if (stat(path, &dir_stat) < 0) {
+      return 0;
+  }
+  return S_ISDIR(dir_stat.st_mode);
+}
+
+inline int exists(const std::string& path) {
```

Comment:
I can see that returning 0 (true) and -1 (false) fits into the existing pattern, but I think it would be better to return a `bool` at least from `is_directory()` and `exists()`.

Reply:
Fixed in [163bbea](https://github.com/apache/nifi-minifi-cpp/pull/942/commits/163bbeaa1f80756ff7d9b3f034e4e0bc71c3dcee)